hbase-client/src/main/java/org/apache/hadoop/hbase/coprocessor/ColumnInterpreter.java

```diff
@@ -40,11 +40,6 @@
  * {@link ColumnInterpreter#castToReturnType(Object)} which takes a &lt;T&gt; type and returns a
  * &lt;S&gt; type. The AggregateIm&gt;lementation uses PB messages to initialize the user's
  * ColumnInterpreter implementation, and for sending the responses back to AggregationClient.
- * @param T Cell value data type
```

Comment:
I think these lines are still useful, let's keep them without using the `@param` tag?